DOCS-6433 edge metrics (#50)
mulesoft/docs-visualizer: modules/ROOT/pages/troubleshoot-visualizer.adoc

|For CloudHub applications, you must use an `-AM` Mule to support Anypoint Monitoring, or upgrade to a Mule 4 version available after March 22, 2019, or Mule 3.9 version available after April 5, 2019, and then enable the Monitoring agent. View a list of xref:setup.adoc[supported Mule versions].
|===

== Why can't I see metrics for my connections?

> Metrics at the connection level are only available for certain types of connection. For more information on edge metrics compatibility, see xref:connection-metrics-compatibility.adoc[Supported Connection-Level Metrics].

Reviewer: certain types of connection**s**

^^ just a minor typo

Author: ```suggestion
Metrics at the connection level are only available for certain types of connections. For more information on edge metrics compatibility, see xref:connection-metrics-compatibility.adoc[Supported Connection-Level Metrics].
```

> Metrics at the connection level are only available for certain types of connection. For more information on edge metrics compatibility, see xref:connection-metrics-compatibility.adoc[Supported Connection-Level Metrics].

Reviewer 2: NOTE "at the connection level". I think we want to change all instances of "on the connection level" to "at". Please search for and evaluate.

> Metrics at the connection level are only available for certain types of connection. For more information on edge metrics compatibility, see xref:connection-metrics-compatibility.adoc[Supported Connection-Level Metrics].

Reviewer 2: ```suggestion
Metrics at the connection level available for only certain types of connections. For more information about edge metrics compatibility, see xref:connection-metrics-compatibility.adoc[Supported Connection-Level Metrics].
```
